src/containers/calendar-container.js

```diff
@@ -79,7 +79,7 @@ const mapDispatchToProps = dispatch => {
             dispatch(page(1));
         },
         showToday: () => {
-            dispatch(setCurrentlyViewingDate(moment()));
+            dispatch(setCurrentlyViewingDate(moment().utcOffset(-480)));
```

Comment:
Probably a temporary fix until we can deal with timezones better across both projects.

Reply:
Add a TODO: or FIXME: so we can find this later.